core/src/com/unciv/ui/screens/worldscreen/bottombar/BattleTable.kt

```diff
@@ -204,9 +204,11 @@ class BattleTable(val worldScreen: WorldScreen) : Table() {
 
         val maxDamageToDefender = BattleDamage.calculateDamageToDefender(attacker, defender, tileToAttackFrom, 1f)
         val minDamageToDefender = BattleDamage.calculateDamageToDefender(attacker, defender, tileToAttackFrom, 0f)
+        val avgDamageToDefender = maxDamageToDefender - ((maxDamageToDefender - minDamageToDefender)/2.0).roundToInt()
```

Comment:
Maybe math had an avg function?

Reply:
Nope , doesn't look like it

Reply:
As far as I am aware there is no functionality for average or mean calculation in java.lang.Math. Does unciv import any other math libraries that I should use?

Reply:
Yes, I understand. I was just wondering if you are using any math libs elsewhere to do averages you could suggest. I just uploaded a new patch switching it to use the collections average if you would prefer it to use functions.